scst-policy/configuring.md

```diff
@@ -1,205 +1,142 @@
-**TODO** update this section
 # Configuring Supply Chain Security Tools - Policy
 
 This component requires extra configuration steps to start verifying your
 container images properly.
 
-The instructions in this section only apply to the deployment namespace of
-Supply Chain Security Tools - Sign. In most cases, this namespace is
-rendered as the default namespace `image-policy-system`.
+## <a id="admission-of-images"></a> Admission of Images
 
-If you deployed Supply Chain Security Tools - Sign by using a customized
-namespace specified in the installation values file, replace `image-policy-system`
-with the namespace name that you specified in `deployment_namespace` before
-performing the configuration steps.
+An image is admitted after it has been validated against all
+[ClusterImagePolicy](#create-cip-resource) that matched the digest of the image
+and that there was at least one valid signature or attestation obtained from the
+authorities provided in each of the matched ClusterImagePolicy. So each
+ClusterImagePolicy that matches is AND for admission, and within each
+ClusterImagePolicy authorities are ON.
+
+## <a id="including-namespaces"></a> Including Namespaces
+
+The cosigned admission controller will only validate resources in namespaces
+that have chosen to opt-in. This can be done by adding the label
+cosigned.sigstore.dev/include: "true" to the namespace resource.
```

Comment:
This can be done by adding the label
`policy.sigstore.dev/include: "true"` to the namespace resource.

I also wanted to add the "`" to the label to make it stand out more.